README.md

```diff
@@ -13,17 +13,21 @@ As of now, `recheck` will give you the following help message:
 ```
 $ recheck
 Usage: recheck [--help] [--version] [COMMAND]
+
+Description:
```

Comment:
Should we separate the description by one more empty line? As [picocli](https://picocli.info/#_expanded_example) does?

Reply:
I decided against it since it looked kind of too 'spacious', what do you prefer?